src/components/ImageView/index.js

```diff
@@ -52,6 +52,15 @@ class ImageView extends PureComponent {
         document.addEventListener('mousemove', this.trackMovement.bind(this));
     }
 
+    componentDidUpdate() {
+        if (this.canUseTouchScreen) {
+            return;
+        }
+        Image.getSize(this.props.url, (width, height) => {
+            this.setImageRegion(width, height);
+        });
+    }
+
```

Comment:
Why this is necessary?

Can you elaborate?

Reply:
I believe so, I noticed in iOS when going through images if one image starts with a certain zoom compared to the others, then this will persist since nothing is handling the change of source in this regard.

Reply:
This is what I'm getting without that code. You'll see there's an image zoomed in and the sides are cut off.

https://user-images.githubusercontent.com/6119181/171294431-d20eb7f4-50d3-4be2-8144-9dde43173f06.mov